config/initializers/json.rb

```diff
@@ -0,0 +1,24 @@
+require 'json/version'
+module JSON
```

Comment:
@ohadlevy - Not sure we need, as we don't override any method. We are adding a new method.
I am adding a comment to remove this method when updating to json ~> 1.8, and will open an issue to remove this when 1.8 is in.